.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: php
+php:
+#  - '5.4' -- breaks for some weird reason (https://travis-ci.org/wikimedia/popularpages/jobs/216541216)
```

Comment:
I changed it. It's still giving an error without any helpful logs. https://travis-ci.org/wikimedia/popularpages/jobs/218174186